src/assets/global.scss

```diff
@@ -1,16 +1,20 @@
 @import "assets/colors.scss";
 @import "assets/variables.scss";
 
+.text-black {
+  color: $color-black !important;
+}
+
 .page-content-container {
   .progress {
-    @extend %progress-styles;
+    @extend %progress-styles !optional;
```

Comment:
What is the purpose of the optional?

Reply:
I had a problem with Sass ( this is a example of it )

> ".ex-class" failed to extend ".class"
The selector ".class" was not found.
This will be an error in future releases of Sass
Use "@extend .class !optional" if the extend should be able to fail


This prevents to blow the page away